glossary/glossary-RnD process.md

```diff
@@ -0,0 +1,32 @@
+# R&D Process Glossary
+
+This table contains terms specifically related to the research and development methodology used within QF Project. These terms describe different stages in the hypothesis-driven development approach, from initial ideas through investigation, testing, implementation, and verification. This table is particularly useful for understanding how new features and improvements are conceptualized, evaluated, and incorporated into the project.
+
+| Term | Abbreviation | Description | Comment |
```

Comment:
Table format seems hard to read and modify in a raw text. What if we make each term a header like in the Polkadot glossary (link below)? Both rendered preview and the raw format look fine.

https://github.com/w3f/polkadot-wiki/blob/ab738976a9c0aec7feeb81d6492347520a82b362/docs/general/glossary.md?plain=1